### PR TITLE
[SAGE TUTORIAL] Integrating Blockchain Technology into Legacy Systems on Celo

### PR DESCRIPTION
**Pull request by iamoracle**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -690,11 +690,11 @@ const Users: User[] = [
   //   tags: ['favorite'],
   // },
   {
-    title: ' Celo Composer',
-    description: 'Build on Celo in 5 minutes of less with Celo Composer.',
-    preview: require('./showcase/celo-composer.png'),
-    website: '/tutorials?tags=composer',
-    tags: ['favorite'],
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:697]: I don't think this is right can you undo the changes from this file

File: src/data-tutorials/users.tsx [hunk 2]
```diff
@@ -747,6 +748,15 @@ const Users: User[] = [
...
   },
+  {
+    title: "Integrating Blockchain Technology into Legacy Systems on Celo",
+    description:
+      "Blockchain technology can facilitate new forms of collaboration and innovation by enabling the exchange of information and value between different parties.",
+    preview: require("./showcase/celo-sage.png"),
+    website:
+      "/blog/tutorials/blockchain-integration-updating-legacy-systems-with-the-celo-platform",
+    tags: ["intermediate", "celo"],
+  },
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:759]: this block is fine.

File: src/data-tutorials/users.tsx [hunk 3]
```diff
@@ -853,18 +864,21 @@ const Users: User[] = [
     tags: ['advanced', 'foundation', 'crowdfunding', 'composer'],
   },
   {
-    title: 'React Native & Celo - Easily build React Native dApps on Celo',
-    description: 'Quickly develop Android and iOS apps on Celo using the Celo Composer for React Native.',
-    preview: require('./showcase/intermediate/easily-build-react-native-dapps-on-celo.png'),
-    website: 'blog/tutorials/React-Native-&-Celo-Easily-build-React-Native-dApps-on-Celo',
-    tags: ['intermediate','foundation', 'reactnative', 'composer'],
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:860]: Don't change the single quotes to double quotes

Comment (iamoracle) [reply src/data-tutorials/users.tsx:860]: Hello @developerkunal, I don't seem to get the problem. Can you make the edit on my behalf?
